gemini/use-cases/using-gemini-with-bigquery-remote-functions/storage.tf

```diff
@@ -42,18 +42,10 @@ resource "google_storage_bucket_object" "text_source_upload" {
 # Create a GCS bucket to upload demo images
 ## Use a random string for the bucket name to avoid conflicts
 resource "google_storage_bucket" "demo_images" {
-  name                        = "gemini-bq-demo-images${random_id.id.hex}"
+  name                        = "gemini-bq-demo-images-${random_id.id.hex}"
   project                     = module.project-services.project_id
   location                    = var.region
   uniform_bucket_level_access = true
   force_destroy               = var.force_destroy
   depends_on                  = [time_sleep.wait_after_apis]
 }
-
-## Upload the sample images to the bucket
-resource "google_storage_bucket_object" "image_upload" {
-  for_each = fileset("${path.module}/src/images", "*.jpg")
-  name     = each.value
-  bucket   = google_storage_bucket.demo_images.name
-  source   = "${path.module}/src/images/${each.value}"
-}
```

Comment:
I guess there's not a way to copy between GCS buckets in terraform?

Thanks for adding the extra steps in Workflows, I didn't realize you'd need to change it that much!